tokenizers/src/models/unigram/model.rs

```diff
@@ -78,18 +81,35 @@ pub enum UnigramError {
 impl Default for Unigram {
     fn default() -> Self {
         let vocab = vec![("<unk>".to_string(), 0.0)];
-        Self::from(vocab, Some(0)).unwrap()
+        Self::from(vocab, Some(0), Some(false)).unwrap()
     }
 }
 
+pub fn byte_to_piece(c: u8) -> String {
+    format!("<0x{:02X}>", c)
+}
+
+pub fn piece_to_byte(piece: &str) -> Option<u8> {
+    let mut k_map: HashMap<String, u8> = HashMap::new();
+    for i in 0..=255 {
+        let byte_piece = byte_to_piece(i);
+        k_map.insert(byte_piece.clone(), i);
+    }
+    k_map.get(piece).copied()
+}
+
```

Comment:
This is not used.

Reply:
Indeed, will add it in another PR for training